src/simulator/api.yaml

```diff
@@ -1778,6 +1778,17 @@ components:
         homeTransactionId:
           type: string
           description: Transaction ID from the DFSP backend, used to reconcile transactions between the switch and DFSP backend systems
+        fulfilment:
+          type: string
+          description: Fulfilment from the DFSP backend, used for testing purposes to inject an invalid fulfilment via a rule.

```

Comment:
Could you also align this to the FSPIOP spec please.

Reply:
New fulfilment fields now references IlpFulfilment per review. Thank you @mdebarros